hydra-node/src/Hydra/Ledger/Simple.hs

```diff
@@ -83,6 +89,15 @@ instance FromCBOR SimpleTx where
       <*> fromCBOR
       <*> fromCBOR
 
+instance HasTextEnvelope SimpleTx where
+  textEnvelopeType _ = "SimpleTx"
+
+instance HasTypeProxy SimpleTx where
+  data AsType SimpleTx = AsSimpleTx
+  proxyToAsType _ = AsSimpleTx
+
+deriving instance SerialiseAsCBOR SimpleTx
```

Comment:
Should: not be required to add these!

These type classes are from `cardano-api` and we like to keep `SimpleTx` non-cardano-specific.